src/components/programs/Program.vue

```diff
@@ -1,68 +1,56 @@
 <template>
-  <router-link class="flex p-10 bg-white border border-solid shadow-sm cursor-pointer hover:bg-gray-50" :to="programUrl">
+  <router-link class="flex rounded-lg p-10 bg-white border border-solid shadow-sm cursor-pointer hover:bg-gray-50" :to="programUrl">
     <div class="flex items-center w-full">
       <div class="flex items-center justify-between w-full">
         <div class="flex flex-col">
           <div class="text-xl">
             {{ id }}
           </div>
           <div class="text-gray-500">
-            3/3 processes
+            {{ runningProcesses }}/{{ processes.length }} process{{ processes.length > 1 ? "es" : "" }}
```

Comment:
We can handle plural forms directly using Vue I18n: https://vue-i18n-next.intlify.dev/guide/essentials/pluralization.html#basic-usage.